resources/views/dashboard/controllers/roster.blade.php

```diff
@@ -1,319 +1,170 @@
-                                @elseif($c->app == 88)
-                                    <td><center style="color:#c1ad13">m</center></td>
-                                @elseif($c->app == 89)
-                                    <td><center style="color:#c1ad13">M</center></td>
-                                @elseif($c->app == 90)
-                                    <td><center><b style="color:blue;">A80 SAT</b></center></td>
-                                @elseif($c->app == 91)
-                                    <td><center><b style="color:blue;">A80 DR</b></center></td>
-                                @elseif($c->app == 92)
-                                    <td><center><b style="color:blue;">A80 TAR</b></center></td> 
-                                @elseif($c->app == 99)
-                                    <td><center><i class="fab fa-stripe-s" data-toggle="tooltip" style="color:#c1ad13" title="Expires: {{ $c->solo }}"></i></center></td>
+                                @if(($c->clt_twr > 0)||($c->twr == 2))
+                                <span class="badge badge-danger">TWR</span>
                                 @endif
-                                @if($c->ctr == 0)
-                                    <td><center><i class="fas fa-times" style="color:red"></i></center></td>
-                                @elseif($c->ctr == 1)
-                                    <td><center><i class="fas fa-check" style="color:green"></i></center></td>
-                                @elseif($c->ctr == 88)
-                                    <td><center style="color:#c1ad13">m</center></td>
-                                @elseif($c->ctr == 89)
-                                    <td><center style="color:#c1ad13">M</center></td>
-                                @elseif($c->ctr == 99)
-                                    <td><center><i class="fab fa-stripe-s" data-toggle="tooltip" style="color:#c1ad13" title="Expires: {{ $c->solo }}"></i></center></td>
+                                @if(($c->clt_app > 0)||($c->app > 1))
+                                <span class="badge badge-info">APP</span>
                                 @endif
+                            </td>
+                            <!-- ATL Tier 1 -->
+                            <td class="text-center">
+                                @if(($c->atl_gnd > 0)||($c->gnd == 2))
+                                <span class="badge badge-primary">DEL</span>
+                                <span class="badge badge-success">GND</span>
+                                @endif
+                                @if(($c->atl_twr > 0)||($c->twr == 2))
+                                <span class="badge badge-danger">TWR</span>
+                                @endif
+                                @if(($c->atl_app > 0)||($c->app > 1))
+                                <span class="badge badge-info">
+                                    @if($c->app == 90)
```

Comment:
Same as above about magic numbers. Having these arbitrary numbers scattered about the codebase is ripe for mistakes... this would be much better if we had a `ATL_SAT_APP` as `90` somewhere. More clear about what it actually means and much less likely to make mistakes.

Reply:
See above.

Reply:
c/w